agents/agents-core/src/commonMain/kotlin/ai/koog/agents/core/agent/AIAgent.kt

```diff
@@ -368,3 +413,16 @@ public open class AIAgent(
 
     //endregion Private Methods
 }
+
+public fun singleRunStrategy(): AIAgentStrategy = strategy("single_run") {
```

Comment:
kdoc?

Reply:
Also, let's please explain that the single-run term does NOT mean that the agent won't run more than one tool iteration.
And probably an example with some trace of some execution (ex: same as what I was showing on the KotlinConf presentation with banking app)